ipaserver/install/installutils.py

```diff
@@ -268,8 +268,11 @@ def add_record_to_hosts(ip, host_name, conf_file=paths.HOSTS):
 def read_ip_addresses():
     ips = []
     print("Enter the IP address to use, or press Enter to finish.")
+    msg = "Please provide the IP address to be used for this host name"
     while True:
-        ip = ipautil.user_input("Please provide the IP address to be used for this host name", allow_empty = True)
+        if ips:
+            msg += " or leave blank to continue with the installation"
```

Comment:
You appending text to message in cycle, `Please provide the IP address to be used for this host name  or leave blank to continue with the installation or leave blank to continue with the installation or leave blank to continue with the installation`